srcpkgs/tuner/template

```diff
@@ -0,0 +1,16 @@
+# Template file for 'tuner'
+pkgname=tuner
+version=1.5.1
+revision=1
+build_style=meson
+hostmakedepends="gettext pkg-config vala"
```

Comment:
Seems `glib-devel` is needed as `hostmakedepends` as well to get the `glib-compile-resources` on cross targets

Reply:
I'm sorry, I'm kinda new to all this... is what I've done OK? :-\\

Reply:
Should be good, with a few remarks:

- any blank lines are unnecessary
- we usually indent wrapped lines like the `makedepends` with only one space

As mentioned below we do one commit per package, so you need to squash all these. In the future, you can use `git commit --amend` to add changes without introducing new commits.

Reply:
> * any blank lines are unnecessary

I believe I had only one blank line at the end... POSIX standard I believe, everything ends with a new line... please correct me if I'm wrong :).

> * we usually indent wrapped lines like the `makedepends` with only one space

As in not do a bunch of spaces to align the indent? OK, no problem, feel free to correct the template and I will remember this for future commits ;).

> As mentioned below we do one commit per package, so you need to squash all these. In the future, you can use `git commit --amend` to add changes without introducing new commits.

I still haven't connected my main rig with GitHub, I'm still working on the web interface :-\\... I know, I'll get on it as soon as possible.

Now, how do I squash the commits :). Might be a stupid question, I know, but as I said, I'm kinda new :).

Reply:
> Now, how do I squash the commits

See "Squashing commits" in https://git-scm.com/book/en/v2/Git-Tools-Rewriting-History, should be pretty clearly described (beware, `git rebase -i` sends you to the default editor, which might be vim on linux 😛)

Reply:
That's easy to solve, don't use the web ui.